src/ui/backend.rs

```diff
@@ -222,9 +222,19 @@ where
         self.terminal.write_styled(&x)
     }
 
-    fn print_option_value<D: Display>(&mut self, option: &ListOption<D>) -> Result<()> {
+    fn print_option_value<D: Display>(
+        &mut self,
+        option: &ListOption<D>,
+        is_selected: bool,
```

Comment:
Instead of comparing `idx` at each function that calls this (3 atm, might grow), let's pass `page: &Page<ListOption<D>>` instead and calculate `is_selected` on this function. Just like the index checks we do on `print_option_prefix`.

Reply:
Done in https://github.com/mikaelmello/inquire/pull/88/commits/8a655ffd4ed3c39da810d8171f79ea5f3b52a163. I had to pass idx also.

Reply:
`ListOption<T>` has the index of the option as a property, so I think you can use that. (I might be misremembering how the idx value was used for ListOptions and be wrong)

Reply:
Done in https://github.com/mikaelmello/inquire/pull/88/commits/c701464b25e398d3b748c15e84ba5e1dc9b8ddbd